src/ui/panels.ts

```diff
@@ -0,0 +1,22 @@
+import * as vscode from 'vscode'
+import * as path from 'path'
+import { PanelType } from './panel'
+
+export const WelcomePanel: PanelType = {
```

Comment:
What is the purpose of the WelconePanel? Do you known extension that can illustrate the use-case?

Reply:
At first that was a *Hello World* code, but some of extension are doing kind of page for major or else.
As I said, It is so far **`“focused on the mecanism rather than the message”`**

Reply:
ok

Reply:
:)